app/assets/javascripts/app/views.js

```diff
@@ -55,8 +55,7 @@ app.views.Base = Backbone.View.extend({
       .html(this.template(presenter))
       .attr("data-template", _.last(this.templateName.split("/")));
 
-    // add avatar fallback if it can't be loaded
-    this.$el.find(this.avatars.selector).on("error", this.avatars.fallback);
+    this.setupAvatarFallback();
```

Comment:
Can we also use this here?
* [app/assets/javascripts/app/views/tags_view.js](https://github.com/diaspora/diaspora/blob/1b9ab16039160aefe16dc255487fb739b97f0a55/app/assets/javascripts/app/views/tags_view.js#L9)
* [app/assets/javascripts/app/views/notification_dropdown_view.js](https://github.com/diaspora/diaspora/blob/1b9ab16039160aefe16dc255487fb739b97f0a55/app/assets/javascripts/app/views/notification_dropdown_view.js#L99)